docs/docs/transactions.md

```diff
@@ -84,11 +91,13 @@ sequelize.transaction(function (t1) {
 });
 ```
 
+### Nested/Partial transactions
```

Comment:
Hmm actually i'm not sure if they automatically nest with CLS, @janmeier will have to answer that.